src/amo/reducers/guides.js

```diff
@@ -0,0 +1,59 @@
+/* @flow */
+import invariant from 'invariant';
+
+import { LOAD_ADDON_RESULTS } from 'core/reducers/addons';
+
+export const FETCH_GUIDES_ADDONS: 'FETCH_GUIDES_ADDONS' = 'FETCH_GUIDES_ADDONS';
+
+export type GuidesState = {|
+  loading: boolean,
+|};
+
+export const initialState: GuidesState = {
+  loading: false,
+};
+
+export type FetchGuidesParams = {|
+  errorHandlerId: string,
+  guids: string,
```

Comment:
Maybe it would be less error prone to use `Array<string>` here and to do `.join(',')` in the saga. That would also communicate the intent better.

Reply:
I started work on the Guide Page itself and I am doing the `join(`) there. I am not sure if it really matters but I guess I can move this over to the saga..

Reply:
it would be better to move it because expecting a string with comma separated values is a weak contract. If it is done in the saga, then you can enforce it only once and callers don't need to know that (the comma separated values become an implementation detail)